python/tests/unit/arcticdb/version_store/test_query_builder.py

```diff
@@ -60,6 +62,41 @@ def test_querybuilder_date_range_then_filter(lmdb_version_store_tiny_segment, us
     assert_frame_equal(expected, received)
 
 
+def test_querybuilder_filter_datetime_with_timezone(lmdb_version_store_tiny_segment):
+    lib = lmdb_version_store_tiny_segment
+    cnt = [0]
+    def can_read_back(write_with_time, filter_with_time):
+        symbol = f"sym_{cnt[0]}"
+        cnt[0] += 1
```

Comment:
nit: No need for a new symbol on each call, can remove this temporary

Reply:
Reusing the same symbol now